packages/deeplinks/search/constants.ts

```diff
@@ -18,6 +18,7 @@ export const SERVERLESS_ES_CONNECTORS_ID = 'serverlessConnectors';
 export const SERVERLESS_ES_WEB_CRAWLERS_ID = 'serverlessWebCrawlers';
 export const ES_SEARCH_PLAYGROUND_ID = 'searchPlayground';
 export const SERVERLESS_ES_SEARCH_INFERENCE_ENDPOINTS_ID = 'searchInferenceEndpoints';
+export const SERVERLESS_ES_SEARCH_SYNONYMS_ID = 'searchSynonyms';
```

Comment:
I think we can drop `SERVERLESS_` since this app id/deep link will be used in both serverless and stack.
```suggestion
export const ES_SEARCH_SYNONYMS_ID = 'searchSynonyms';
```